01_qscintilla.pro.dpatch by Torsten Marek <shlomme@debian.org>
Don't install the library directly
Index: qscintilla2-2.6.2/Qt4Qt5/qscintilla.pro
===================================================================
--- qscintilla2-2.6.2.orig/Qt4Qt5/qscintilla.pro	2012-06-20 14:09:15.000000000 -0400
+++ qscintilla2-2.6.2/Qt4Qt5/qscintilla.pro	2012-07-10 01:19:47.348028182 -0400
@@ -42,7 +42,7 @@
 #DEFINES += SCI_NAMESPACE
 
 # Handle both Qt v4 and v3.
-target.path = $$[QT_INSTALL_LIBS]
+target.path = /usr/lib
 isEmpty(target.path) {
 	target.path = $(QTDIR)/lib
 }
